infrastructure/resource_group_level/modules/storage.bicep

```diff
@@ -189,6 +189,16 @@ resource portalFileShare 'Microsoft.Storage/storageAccounts/fileServices/shares@
   }
 }
 
+resource utilisationReportsFileShare 'Microsoft.Storage/storageAccounts/fileServices/shares@2022-09-01' = {
+  parent: defaultFileService
+  name: 'utilisation-reports'
+  properties: {
+    accessTier: 'TransactionOptimized'
```

Comment:
Any particular reason why you are going with TransactionOptimised rather than Hot?
https://learn.microsoft.com/en-us/azure/storage/files/storage-files-planning#storage-tiers

I doubt it makes much difference!

Reply:
No reason, other than that being used for the other file shares

Reply:
It might be worth a more general tech-debt ticket.
How often are the files accessed? https://learn.microsoft.com/en-us/azure/storage/files/understanding-billing#differences-in-standard-tiers